src/Umbraco.Forms.Integrations.Commerce.EMerchantPay/App_Plugins/UmbracoForms.Integrations/Commerce/eMerchantPay/field-picker.controller.js

```diff
@@ -21,4 +21,4 @@
 }
 
 angular.module("umbraco")
-    .controller("UmbracoForms.Integrations.Commerce.eMerchantPay.FieldPickerController", FieldPickerController);
\ No newline at end of file
+    .controller("UmbracoForms.Integrations.Commerce.emerchantpay.FieldPickerController", FieldPickerController);
```

Comment:
Just wondering whether in our own code we can be consistent with `Emerchantpay`.  This one for example does look a bit odd all lower-case so not sure branding should trump code conventions.  A keep lower-case just for consumer visible names.